spec.emu

```diff
@@ -66,17 +108,31 @@ markEffects: true
             1. Perform ? DefinePropertyOrThrow(_O_, *"stack"*, PropertyDescriptor { [[Value]]: _string_, [[Writable]]: true, [[Enumerable]]: true, [[Configurable]]: true }).
           1. Else,
             1. Assert: _strategy_ is ~accessor~.
-            1. Let _getterClosure_ be a new Abstract Closure with no parameters that captures _string_ and performs the following steps when called:
-              1. Return NormalCompletion(_string_).
-            1. Let _getter_ be CreateBuiltinFunction(_getterClosure_, 0, "", « »).
-            1. Let _setterClosure_ be a new Abstract Closure with parameters (_value_) that captures _O_ and performs the following steps when called:
-              1. Perform ? DefinePropertyOrThrow(_O_, *"stack"*, PropertyDescriptor { [[Value]]: _value_, [[Writable]]: true, [[Enumerable]]: true, [[Configurable]]: true }).
-              1. Return NormalCompletion(*undefined*).
-            1. Let _setter_ be CreateBuiltinFunction(_setterClosure_, 1, "", « »).
-            1. Perform ? DefinePropertyOrThrow(_O_, *"stack"*, PropertyDescriptor { [[Get]]: _getter_, [[Set]]: _setter_, [[Enumerable]]: false, [[Configurable]]: true }).
+            1. If ? IsExtensible(_O_) is *false*, throw a *TypeError* exception.
+            1. Define a new [[ErrorStack]] internal slot on _O_.
+            1. Perform ? DefinePropertyOrThrow(_O_, *"stack"*, PropertyDescriptor { [[Get]]: %ErrorCaptureStackTraceGetter%, [[Set]]: %ErrorCaptureStackTraceSetter%, [[Enumerable]]: false, [[Configurable]]: true }).
```

Comment:
I just realized that the accessor is not enumerable where the data property is, any reason to have this difference?